package.json

```diff
@@ -6,75 +6,67 @@
   "module": "index.mjs",
   "main": "index",
   "files": [
+    "src",
     "types",
     "compiler.*",
     "register.js",
-    "index.*",
-    "ssr.*",
-    "internal",
-    "store",
-    "animate",
-    "transition",
-    "easing",
-    "motion",
-    "action",
-    "elements",
+    "index.d.ts",
+    "internal.d.ts",
+    "store.d.ts",
+    "animate.d.ts",
+    "transition.d.ts",
+    "easing.d.ts",
+    "motion.d.ts",
+    "action.d.ts",
+    "elements.d.ts",
```

Comment:
Are these needed in addition to the exports map because of the moduleResolution thing?

Reply:
Correct